package.json

```diff
@@ -27,6 +27,7 @@
     "mocha-server": "node bin/mocha-server.js",
     "firefox-unit-test": "node bin/firefox-driver --test",
     "firefox": "node bin/firefox-driver --start",
+    "chrome": "node bin/chrome-driver",
```

Comment:
we should probably remove chrome/node from the docs as we are no longer working on supporting them.